### ISSUE TITLE
MPV instances continue playback when their windows NOT in focus and even BEFORE users authorized after PC woke after sleep

### ISSUE THREAD

**Issue by DRSAgile** [mpv-player/mpv#11458]
### Important Information
- mpv version 0.35.0-317g-9880b06a
- Windows Version Windows 11 22H2

### Reproduction steps

1. Add to `input.conf` the following:
```
MBTN_LEFT cycle pause
SPACE cycle pause
```
2. Open one or more videos in MPV, leave it unplayed/in pause
3. Put PC to sleep
4. wake it up

### Expected behaviour

Nothing happening with MPV

### Actual behaviour

Every instance of MPV starts playback of the videos. Then, after the authorization is complete, you have to stop the playback in each instance of MPV.

The question is how to force MPV instances to not autostart playback after waking up the PC, let alone when the player is out of focus and the user did not even log in the shell yet.

### Log file

[Log file](https://disk.yandex.ru/d/VDGVR37L6u_1Kg)

### Sample files

Not applicable.

**Comment by DRSAgile** [comment]
What does "ignored-issue-template" mean?

No log or sample files make sense in the issue.

**Comment by CounterPillow** [comment]
> Log file
> Not applicable.

Yes applicable. That's what it means. It tells us what events mpv received.

**Comment by DRSAgile** [comment]
I could not so far reproduce the exact behaviour in the issue with the logging turned on, so I am not editing the root post.

However, [I did collect a log](https://disk.yandex.ru/d/14w-RRPZZ2dRWw) with some warnings/errors that might be related to the issue.

The excerpts from the log in question:

```
[ffmpeg/audio] mp3float: Header missing
Error decoding audio.
```

Despite this error, the video is played back with audio just fine.

```
Audio device underrun detected.
```
This one caused the player to freeze and an audio device reconnection.

So, it might be that this what actually happens -- at least, sometimes -- during waking up, but I am not sure how it should/would cause playback to start in the background (before log on).
